docs/dev/reference/dca/list.md

```diff
@@ -162,7 +163,7 @@ $GLOBALS['TL_DCA']['tl_example']['list']['operations'] = [
 {{% notice "info" %}}
 {{< version-tag "5.5" >}}
 All operations are now shown within the context-menu. If you want them to appear in the overview, you can enable them by
-prepending your key with `!` as seen in the example. Note that the operations `edit`, `children` and `toggle` will always appear in
+prepending your key with `!` as seen in the example (or use `'primary' => true` in your custom operation config). Note that the operations `edit`, `children` and `toggle` will always appear in
```

Comment:
I think this (original) text is misleading, because the exclamation mark only works for named/default operations.